docs/source/release_notes.rst

```diff
@@ -38,6 +39,12 @@ Release Notes
         * Renamed ``LabelLeakageDataCheck`` to ``TargetLeakageDataCheck`` :pr:`1319`
 
 
+.. warning::
```

Comment:
@angela97lin can you merge this with the breaking changes above? I think this got missed during branch update

Reply:
Yup! Updating docs and cleaning up some other doc stuff right now. Will update when this is good to go!

Reply:
Docs look good: https://evalml.alteryx.com/en/1201_max_pipelines_removal/release_notes.html